src/EnergyPlus/Plant/PlantManager.cc

```diff
@@ -4348,8 +4344,8 @@ namespace EnergyPlus {
                     ShowWarningError("Plant Loop: " + PlantLoop(LoopNum).Name +
                                      " Demand Side is storing excess heat the majority of the time.");
                 }
-                if (PlantLoop(LoopNum).LoopSide(DemandSide).LoopSideInlet_CapExcessStorageTime >
-                    PlantLoop(LoopNum).LoopSide(DemandSide).LoopSideInlet_TotalTime / 2) {
+                if (PlantLoop(LoopNum).LoopSide(SupplySide).LoopSideInlet_CapExcessStorageTime >
+                    PlantLoop(LoopNum).LoopSide(SupplySide).LoopSideInlet_TotalTime / 2) {
```

Comment:
The C++ compiler should've understood our intention here and told us to put `SupplySide` instead.